openwisp_monitoring/device/static/monitoring/css/netjsongraph.css

```
@@ -1,64 +1,209 @@
 }
-.njg-container .unknown div {
-  background-color: rgba(53, 60, 68, 0.8) !important;
+.njg-container .njg-date {
+  position: absolute;
+  right: 20px;
+  bottom: 20px;
+  z-index: 1;
+}
+
+@font-face {
+  font-family: "iconfont";
```

Stylelint (17.14.0)

[error] Expected no quotes around "iconfont" (font-family-name-quotes)

(font-family-name-quotes)